helm_deploy/prisoner-content-hub-frontend/templates/_envs.tpl

```diff
@@ -46,6 +46,9 @@ env:
           name: {{ .Values.application.contentConfigMapName }}
           key: internalUrl
 
+   - name: APPROVED_VISITORS_FEATURE_ENABLED
```

Comment:
The single space might cause it to break, it's very picky 😄

Reply:
updated.